project/frontend/src/components/ResultsMap.js

```diff
@@ -10,8 +10,8 @@ const mapStyle = {
 function MapContainer(props) {
   const [activeMarker, changeMarker] = useState({});
   const [showInfoWindow, changeInfoVisibility] = useState(false);
-
-  const coords = { lat: 40.837, lng: -73.865 };
+  const restaurant = props.restaurant;
+  const matchCoords = restaurant[0].key.latLngCoords;
```

Comment:
Even though we check for an empty list in getRecommendation, I think it might be good to do a check here just to be safe so that we don't get an index out of bounds. Jesse suggested that we make a component called NoResults or something, so we could just redirect to that if restaurat.length === 0

Reply:
I guess we can add that as a todo and right now you could just alert the user and return or something?

Reply:
going to add this as a TODO as discussed in our sync today!